releasenotes/notes/trace-agent-evp-forwarding-api-000a992cbfe7c14b.yaml

```diff
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    Add support for new types of CI Visibility payloads to the Trace Agent, so
```

Comment:
Not sure if this applies here but there was an unwritten rule at some point that any trace-agent change gets prefixed with `APM: `. Frankly I'm not sure it makes sense in this case.

Reply:
I didn't know that rule, but I don't think this is really "APM" anymore so I would prefer leaving it like this.